trade-finance-manager-api/src/v1/api.js

```diff
@@ -4,14 +4,8 @@ const { isValidMongoId, isValidPartyUrn, isValidNumericId, isValidCurrencyCode,
 const CONSTANTS = require('../constants');
 require('dotenv').config();
 
-const {
-  DTFS_CENTRAL_API_URL,
-  EXTERNAL_API_URL,
-  DTFS_CENTRAL_API_KEY,
-  EXTERNAL_API_KEY,
-  AZURE_ACBS_FUNCTION_URL,
-  AZURE_NUMBER_GENERATOR_FUNCTION_URL
-} = process.env;
+const { DTFS_CENTRAL_API_URL, EXTERNAL_API_URL, DTFS_CENTRAL_API_KEY, EXTERNAL_API_KEY, AZURE_ACBS_FUNCTION_URL, AZURE_NUMBER_GENERATOR_FUNCTION_URL } =
+  process.env;
```

Comment:
I'm guessing the formatter did this? Surprising that it re-formatted it in this way 😅